frontend/public/locales/en/renew-child.json

```diff
@@ -279,5 +279,89 @@
       "sin-valid": "Must be a valid SIN",
       "sin-unique": "The Social Insurance Number (SIN) must be unique"
     }
+  },
+  "confirm-address": {
+    "page-title": "Address",
+    "have-you-moved": "Would you like to update your home or mailing address?",
+    "is-home-address-same-as-mailing-address": "Is your home address the same as your mailing address?",
+    "radio-options": {
+      "yes": "Yes",
+      "no": "No"
+    },
+    "help-message": "The address we have on file can also be found on the renewal letter you received from Service Canada.",
+    "error-message": {
+      "has-address-changed-required": "Select whether you have moved or changed your mailing address",
+      "is-home-address-same-as-mailing-address-required": "Select whether or not your mailing address and home address are the same"
+    },
+    "back-btn": "Back",
+    "continue-btn": "Continue",
+    "cancel-btn": "Cancel",
+    "save-btn": "Save"
+  },
+  "update-address": {
+    "mailing-address": {
+      "page-title": "Mailing address"
+    },
+    "home-address": {
+      "page-title": "Home address",
+      "use-mailing-address": "My mailing address is the same as my home address"
+    },
+    "address-field": {
+      "address": "Address",
+      "address-note": "Include apartment number (if applicable), street number, street name. For example: 123 Main Street Suite 4B",
+      "country": "Country",
+      "province": "Province, territory, state, or region",
+      "city": "City or town",
+      "postal-code": "Postal code or ZIP code",
+      "postal-code-optional": "Postal code or ZIP code (optional)",
+      "select-one": "Select one"
+    },
+    "dialog": {
+      "address-suggestion": {
+        "address-selection-legend": "Please select an address option below:",
+        "cancel-button": "Cancel",
+        "description": "There's a problem with the address you provided.  Please select the address you want to use. If you live in an apartment or suite, make sure to include that information.",
+        "entered-address-option": "Entered address:",
+        "header": "Verify your address",
+        "suggested-address-option": "Suggested address:",
+        "use-selected-address-button": "Use selected address"
+      },
+      "address-invalid": {
+        "close-button": "Close",
+        "description": "The mailing address you entered appears to be invalid. Please review the details below.",
+        "entered-address": "Entered address:",
+        "header": "Invalid address",
+        "use-entered-address-button": "Use entered address"
+      }
```

Comment:
FYI the French content for an invalid address doesn't match the Figma. 
![image](https://github.com/user-attachments/assets/4ade3878-5caa-4274-8366-46732c5c561d)

The invalid address dialog will appear locally (assuming the WSAddress mock is enabled) if we enter a Quebec address:
![image](https://github.com/user-attachments/assets/e10d4059-2fcc-4813-a970-1ec0b95f937f)


(same for French)

Reply:
Updated the en/fr text for validation and also added in review-parent-information.tsx (to help with route errors/invariants)